Payments/Operation/04_ManageCancellationsChargebacks.md

```diff
@@ -0,0 +1,89 @@
+[!!LIST (Cancellations and chargebacks](../UserInterface/05a_ListCancellationsChargebacks.md)
+
+# Monitor cancellations and chargebacks
+
+In the context of a payment transaction a chargeback is a retroactive cancellation of a capture. It is initialized by the customer, for example, if he or she withdraws the payment. A cancellation is a revocation of the reservation of a product that has not yet been delivered. It is also initialized by the customer.    
+
+> [Info] The most payment service providers create a new reference, when a payment has been cancelled or charged back by the customer. However, the most payment service providers support an original reference ID that you can use to search for a specific case in the whole payment process.  
+
+You can use this procedure to display all incoming references to cancellations and chargebacks from a payment service provider. You can check for communication errors between Actindo and the payment service provider that has been caused in a termination of a workflow, for example. In addition, if applicable, you can manually change the status of a transaction to *void* to clear the database.    
+   
+If everything works fine with the posting of a cancellation and chargeback, you do not need to monitor the *Cancellations and chargebacks* menu entry.  
```

Comment:
"If everything works fine..." -> Vielleicht eher "Usually, you do not have to monitor/check, as the workflow manages the payments/payment processing automatically... (Für mich hat "if everything works fine" eine negative Konnotation, daher die Umformulierung, aber vielleicht ist das nur eine persönliche Wahrnehmung).

Reply:
Works fine hört sich nett an